core/src/test/java/org/hibernate/ogm/backendtck/associations/order/OrderByQualifierTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.backendtck.associations.order;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import javax.persistence.OrderBy;
+
+import org.hibernate.ogm.utils.OgmTestCase;
+
+import org.junit.Test;
+
+/**
+ * Tests the use of {@link OrderBy} qualifier, on load association list.
+ *
+ * @author Fabio Massimo Ercoli
+ */
+public class OrderByQualifierTest extends OgmTestCase {
+
+	public static final String CONTRIBUTOR1 = "unclebob";
+	public static final String CONTRIBUTOR2 = "ddalto";
+	public static final String CONTRIBUTOR3 = "telegraph4eternity";
+	public static final String CONTRIBUTOR4 = "fax4ever";
+	public static final String CONTRIBUTOR5 = "phone4induction";
+
+
+	public static final String PROJECT_NAME = "Hibernate OGM";
+
+	@Test
+	public void testOrderByQualifier() {
+		OpenSourceProject project = new OpenSourceProject();
+		project.setName( PROJECT_NAME );
+
+		Programmer unclebob = new Programmer( CONTRIBUTOR1 );
```

Comment:
I think a bit of consistency here ogoes a long way, it's hard for me to remember that the `CONTRIBUTOR1` is `unclebob`, `CONTRIBUTOR2` is `ddalto` and so on. Wouldn't be easier to call them `UNCLEBOB_ID`,` DDALTO_ID` and so on?
 
I mentioned this before but in this case I would make the whole entities as constant.